src/main/java/net/openhft/chronicle/jlbh/JLBH.java

```diff
@@ -332,15 +341,31 @@ private void checkSampleTimeout() {
 
     /**
      * Call this instead of start if you want to install JLBH as a handler on your event loop thread
+     * @deprecated call {@link #eventLoopHandler(EventLoop)}
      */
+    @Deprecated(/* to be removed in x.23 */)
     public JLBHEventHandler eventLoopHandler() {
         if (!jlbhOptions.accountForCoordinatedOmission)
             throw new UnsupportedOperationException();
-        long warmupStart = initStartOSJitterMonitorWarmup();
+        initStartOSJitterMonitor();
+        long warmupStart = warmup();
         warmupComplete(warmupStart);
         return new JLBHEventHandler();
     }
 
+    /**
+     * Call this instead of {@link #start()} if you want to install JLBH as a handler on your event loop thread
+     */
+    public void eventLoopHandler(@NotNull EventLoop eventLoop) {
+        if (!jlbhOptions.accountForCoordinatedOmission)
+            throw new UnsupportedOperationException();
+        initStartOSJitterMonitor();
+        eventLoop.addHandler(new WarmupHandler());
+        Jvm.pause(100);
+        warmupComplete(System.currentTimeMillis());
```

Comment:
I feel like `warmupComplete` could be named better, maybe `waitForWarmupToComplete` or something?